# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/WebrtcImageBuffer.h
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/media/webrtc/libwebrtcglue/WebrtcImageBuffer.h
--- a/dom/media/webrtc/libwebrtcglue/WebrtcImageBuffer.h	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/media/webrtc/libwebrtcglue/WebrtcImageBuffer.h	Fri Oct 23 14:02:54 2020 -0400
@@ -7,8 +7,8 @@
 #ifndef WebrtcImageBuffer_h__
 #define WebrtcImageBuffer_h__
 
-#include "webrtc/common_video/include/video_frame_buffer.h"
-#include "webrtc/rtc_base/keep_ref_until_done.h"
+#include "common_video/include/video_frame_buffer.h"
+#include "rtc_base/keep_ref_until_done.h"
 
 namespace mozilla {
 namespace layers {